### DISCUSSION TITLE
QCarousel and SEO

### DISCUSSION THREAD

**Discussion by DocBouCou** [quasarframework/quasar#11643] [General - Components / Directives / etc]
Hi everyone,

As I am using Quasar for my app, I am also using Quasar for the website introducing my app. This website should be as much optimized for seo as possible.

I am using the carousel component to show up testimonials. The _keep-alive_ prop is set to true.
I have noticed that not all texts are visible in the code (even though I am using keep-alive). So the bots from google, and others, have to use javascript to see and process all the text in my qCarousel. However, Google bots choose to use resources to process javascript only if it "thinks" that it is worth it (thus it is safer to consider that google bots do not really process every javascript hidden stuff).

So, here is my question: how would you do to get a seo-friendly carousel/qCarousel? 
Same question also applies to qMenu (for links in toolbar for instance, links are hidden and consequently not always processed by google).

**Comment by rahu2424** [comment]


**Reply by MilosPaunovic** [reply]
Not related; should be removed.

**Comment by hawkeye64** [comment]
> Not related; should be removed.

The next time that happens please tag someone from the team.
